Update VSCodium icon

cavcrosby opened the pull request:
Fulfills #3401.

SmartFinn commented:
Thanks for the contribution. There are a couple issues with this icon:
1. Missing margins, without it the icons feel much bigger than others. See [Basic concepts](https://github.com/PapirusDevelopmentTeam/papirus-icon-theme/blob/master/tools/work/README.md#basic-concepts) and templates at [tools/work/Papirus/apps](https://github.com/PapirusDevelopmentTeam/papirus-icon-theme/tree/master/tools/work/Papirus/apps)
2. Papirus icons shouldn't use any gradient.

cavcrosby commented:
I believe I have addressed the points you made @SmartFinn.

Will add that I decided to make the icon color blue based on what's given in the example colors SVG (`tools/work/examples-papirus.svg`), I think this color is brighter and "pops" better in appearance.

SmartFinn commented:
Sorry for the late reply.

There is one major issue with these icons, edges of objects should be snapping to the pixel grid. 48px icon is mostly ok, but others icons, especially the 16-32px are blurry.

Here are a couple examples where is ok, and where is bad:

![image](https://user-images.githubusercontent.com/819186/230863238-773fef09-4b78-4db3-82fa-c38bef080eb5.png)
![image](https://user-images.githubusercontent.com/819186/230864618-74d3aeba-afa2-463a-b5b2-ed15b4398558.png)
![image](https://user-images.githubusercontent.com/819186/230864862-69287342-a166-4304-93c2-d988ba9eb5cd.png)

hervyqa commented:
@cavcrosby great work!.
i prefer to keep the white circle, but inside it with the new blue logo.

cavcrosby commented:
In this latest attempt, I've tried to pixel align the edges of objects to the grid in some manner, but unfortunately the geometric shaping of the icon makes perfecting pixel alignment about impossible. If that isn't acceptable, then I can close the PR and let someone else take a crack at it.

SmartFinn commented:
@cavcrosby thank you for your patience :smile: It looks a good enough. I'll merge it when I'll have time to fix minor issues.
